src/components/ClaimableBalances.tsx

```diff
@@ -50,15 +71,17 @@ export const ClaimableBalances = () => {
   return (
     <div className="ClaimableBalances DataSection">
       <Layout.Inset>
-        <Heading2>Claimable Balances</Heading2>
+        <Heading2> Claimable Balance(s)</Heading2>
 
         <Table
```

Comment:
How do you feel about making the table look more like this? Maybe we could even shorten the action to “Claim”. 🤔

![image](https://user-images.githubusercontent.com/7346473/151003293-5304cf21-8bfd-4dfb-97e3-4359620e5ac3.png)

Reply:
This does feel much cleaner, I have implemented this change, thank you